cisco_aci/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Fixed***:
+
+* Bump cryptography to 41.0.3 ([#15517](https://github.com/DataDog/integrations-core/pull/15517))
```

Comment:
```suggestion
* Bump cryptography to 41.0.3. See [#15517](https://github.com/DataDog/integrations-core/pull/15517).
```
Small suggestion to match formatting of past entries.

Reply:
We are changing the format for the changelog. We will update the mast entries to keep things consistent